Sources/_CryptoExtras/RSA/RSA_boring.swift

```diff
@@ -315,6 +345,95 @@ extension BoringSSLRSAPublicKey {
             return output
         }
 
+        fileprivate func blind<H: HashFunction>(
+            _ message: _RSA.BlindSigning.PreparedMessage,
+            parameters: _RSA.BlindSigning.Parameters<H>
+        ) throws -> _RSA.BlindSigning.BlindingResult {
+            let rsaPublicKey = CCryptoBoringSSL_EVP_PKEY_get0_RSA(self.pointer)
+            let modulusByteCount = Int(CCryptoBoringSSL_RSA_size(rsaPublicKey))
+            let e = try ArbitraryPrecisionInteger(copying: CCryptoBoringSSL_RSA_get0_e(rsaPublicKey))
+            let n = try ArbitraryPrecisionInteger(copying: CCryptoBoringSSL_RSA_get0_n(rsaPublicKey))
```

Comment:
I tried this out earlier and it didn't seem great because each of these computed properties would need to to use `PKEY_get0_RSA()` first, which led to a lot of duplicated calls, because we store an `EVP_PKEY *`, rather than an `RSA *` as the backing pointer.

I think these lines are pretty good for local reasoning as none of them escape a pointer beyond the rvalue used in `ArbitraryPrecisionInteger(copying:)`.